frontend/fingertips-frontend/playwright/page-objects/pageFactory.ts

```diff
@@ -2,11 +2,13 @@
 import AxeBuilder from '@axe-core/playwright';
 import SearchPage from './pages/searchPage';
 import ResultsPage from './pages/resultsPage';
+import ChartPage from '@/playwright/page-objects/pages/chartPage';
```

Comment:
The structure of this import is different to the other pages being imported:
```suggestion
import ChartPage from ./pages/chartPage';
```